src/__mocks__/reportTable.mock.ts

```diff
@@ -60,6 +60,7 @@ export const mockReportTableResponse: ReportTableFetchResponse = {
               creation: '2022-05-20T20:00:00+11:00',
             },
           ],
+          performer: [{display: 'Super Man (Identifier: superman)'}],
```

Comment:
when you add such mock data the current test would continue to work and no-one would know that the round brackets are an important consideration here that gets removed in the getRequestor function. It also took me a while to understand where the getRequestor is being testing and what are the requirements. Can you refactor this to articulate better in the test?

Reply:
Added a test to check only provider name is displayed